main.go

```diff
@@ -14,14 +15,19 @@ import (
 //go:embed all:bundle
 var bundle embed.FS
 
+//go:embed schemas/regal-ast.json
+var schema []byte
+
 func main() {
 	// Remove date and time from any `log.*` calls, as that doesn't add much of value here
 	// Evaluate options for logging later..
 	log.SetFlags(0)
 
-	// The embedded FS can't point to parent directories, so while not pretty, we'll
-	// need to pass it from here to the next command
-	cmd.EmbedBundleFS = bundle
+	// Embedded files and directories can only traverse down in the tree, i.e. no parent paths,
+	// so while this isn't pretty, we'll use a separate package to carry state from here. If you
+	// know of a better way of doing this, don't hesitate to fix this.
+	embeds.EmbedBundleFS = bundle
+	embeds.ASTSchema = schema
```

Comment:
thought: hmm, yeah. I don't know of a better way to do this. I think it's best to have the bundle at the top level. I wonder if schema needs to be too though? Perhaps it could sit under `embeds`

Reply:
True. I'll leave it as is for now, but I might move the schema into embeds later 👍